IndexFollowingIT#testFollowIndexWithConcurrentMappingChanges fails

**Build scan**: https://gradle-enterprise.elastic.co/s/7zvpy3v4eu3ok

**Repro line**:

```
./gradlew ':x-pack:plugin:ccr:internalClusterTest' --tests "org.elasticsearch.xpack.ccr.IndexFollowingIT.testFollowIndexWithConcurrentMappingChanges" \
  -Dtests.seed=A15C3DF4E9421E31 \
  -Dtests.security.manager=true \
  -Dtests.locale=uk-UA \
  -Dtests.timezone=Africa/Ndjamena \
  -Druntime.java=11
```

**Reproduces locally?**: no

**Applicable branches**: 

* `master`
* `7.x`

**Failure history**: Failed [8 times within the last 6 months](https://build-stats.elastic.co/app/kibana#/discover?_g=(refreshInterval:(pause:!t,value:0),time:(from:now-6M,mode:quick,to:now))&_a=(columns:!(_source),index:e58bf320-7efd-11e8-bf69-63c8ef516157,interval:auto,query:(language:lucene,query:'project:elasticsearch%20AND%20test:%22testFollowIndexWithConcurrentMappingChanges%22%20AND%20class:%22org.elasticsearch.xpack.ccr.IndexFollowingIT%22%20AND%20stacktrace:%22java.lang.AssertionError:%20timed%20out%20waiting%20for%20green%20state%22'),sort:!(time,desc))) (two times out of this on a PR build)

**Failure excerpt**:

```
java.lang.AssertionError: timed out waiting for green state
	at __randomizedtesting.SeedInfo.seed([A15C3DF4E9421E31:862EA90C40E1F715]:0)
	at org.junit.Assert.fail(Assert.java:88)
	at org.elasticsearch.xpack.CcrIntegTestCase.ensureColor(CcrIntegTestCase.java:338)
	at org.elasticsearch.xpack.CcrIntegTestCase.ensureFollowerGreen(CcrIntegTestCase.java:312)
	at org.elasticsearch.xpack.CcrIntegTestCase.ensureFollowerGreen(CcrIntegTestCase.java:307)
	at org.elasticsearch.xpack.ccr.IndexFollowingIT.testFollowIndexWithConcurrentMappingChanges(IndexFollowingIT.java:289)
```

--- a/x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/IndexFollowingIT.java
+++ b/x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/IndexFollowingIT.java
@@ -239,7 +239,6 @@ public class IndexFollowingIT extends CcrIntegTestCase {
         }
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/59273")
     public void testFollowIndexWithConcurrentMappingChanges() throws Exception {
         final int numberOfPrimaryShards = randomIntBetween(1, 3);
         final String leaderIndexSettings = getIndexSettings(numberOfPrimaryShards, between(0, 1));
@@ -279,7 +278,9 @@ public class IndexFollowingIT extends CcrIntegTestCase {
                         leaderClient().admin().indices().prepareFlush("index1").setForce(true).get();
                     }
                 }
-                leaderClient().admin().indices().prepareFlush("index1").setForce(true).setWaitIfOngoing(true).get();
+                if (between(0, 100) < 20) {
+                    leaderClient().admin().indices().prepareFlush("index1").setForce(false).setWaitIfOngoing(false).get();
+                }
             }
         });
         thread.start();
@@ -290,7 +291,7 @@ public class IndexFollowingIT extends CcrIntegTestCase {
         ensureFollowerGreen("index2");
 
         for (int i = 0; i < firstBatchNumDocs; i++) {
-            assertBusy(assertExpectedDocumentRunnable(i));
+            assertBusy(assertExpectedDocumentRunnable(i), 1, TimeUnit.MINUTES);
         }
 
         final int secondBatchNumDocs = randomIntBetween(2, 64);
@@ -301,7 +302,7 @@ public class IndexFollowingIT extends CcrIntegTestCase {
         }
 
         for (int i = firstBatchNumDocs; i < firstBatchNumDocs + secondBatchNumDocs; i++) {
-            assertBusy(assertExpectedDocumentRunnable(i));
+            assertBusy(assertExpectedDocumentRunnable(i), 1, TimeUnit.MINUTES);
         }
 
         isRunning.set(false);

